framework/include/materials/Material.h

```diff
@@ -231,7 +231,7 @@ class Material :
   std::set<std::string> _supplied_props;
 
   /// If False MOOSE does not compute this property
-  bool _compute;
+  const bool & _compute;
```

Comment:
I did this to allow for it to be used in a Control, but now that I think about it a bit. You couldn't control the computation because theses objects are stored in different warehouses up front, so changing this flag with a control wouldn't work anyway. So, I will change it back.